packages/flutter_test/test_fixes/flutter_test/animation_sheet_builder.dart

```diff
@@ -0,0 +1,27 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
```

Comment:
To consider: In the framework we kept these tests short and totally focused on the deprecated API without fluff. So, this would just be:

```
final AnimationSheetBuilder animationSheet = AnimationSheetBuilder(frameSize: const Size(48, 24));
final Widget display = await animationSheet.display();
final Widget display2 = await animationSheet.display(key: UniqueKey());
```

I find that this makes these tests easier to review because you don't have to hunt for the differences.

Reply:
Shortened this test and a few of the others.